pkg/driver/driver.go

```diff
@@ -72,8 +72,8 @@ type Driver interface {
 }
 
 type BaseDriver struct {
-	Instance *store.Instance
-	Yaml     *limayaml.LimaYAML
+	Instance   *store.Instance
+	InstConfig *limayaml.LimaYAML
```

Comment:
So this is outside the scope of this PR, but I'm wondering why the `BaseDriver` has an `InstConfig` field. Isn't that a duplicate of `Instance.Config`?

I don't have time to drill into the code right now, but could `InstConfig` (nee `Yaml`) be replaced with `Instance.Config` everywhere? If not, then we should have a comment explaining when/why they can be different.

@balajiv113 Do you remember why you needed both?